code/modules/mob/living/carbon/human/species/station/station_vr.dm

```diff
@@ -379,7 +379,7 @@
 	wikilink="https://www.yawn.ocry.com/Diona"
 
 /datum/species/teshari
-	mob_size = MOB_MEDIUM
+	mob_size = MOB_SMALL //YW Edit: changed from MOB_MEDIUM to MOB_SMALL
```

Comment:
This is interesting but this was actually the original code upstream of the mob size..... Wonder when it was changed to medium.

Reply:
Virgo did it a long time ago to have all holders use their own version, which relies on the scale instead of mob_size to pick up micros and such instead of the mob_size system.
The issue with reverting to old system by making the teshari mob_size back to small is that a giant scale 200 teshari could be picked up by a scale 25 whatever probably. I'd have to add a scale check where the mob_size check is done to pick up or deny picking up a mob using old system.